README.rst

```diff
@@ -83,6 +83,7 @@ New in Connexion 2.0:
 - Array parameter deserialization now follows the Swagger 2.0 spec more closely.
   In situations when a query parameter is passed multiple times, and the collectionFormat is either csv or pipes, the right-most value will be used.
   For example, `?q=1,2,3&q=4,5,6` will result in `q = [4, 5, 6]`.
+- The spec validator library has changed from `swagger-spec-validator` to `openapi-spec-validator`.
```

Comment:
Please add a note that exceptions changed. I also think Connexion should abstract this exceptions by raising its own `ValidationError` so we can change libraries in the future without breaking code that depends on them.

Reply:
ok, do you think I should do that in this PR, or a different one?

Reply:
(abstract the exception)

Reply:
I think it should be done here.

Reply:
:+1: